docs/content/docs/guide/privaterepo.md

```diff
@@ -18,6 +18,12 @@ This secret contains a [Git Config](https://git-scm.com/docs/git-config) file:
 file: .git-credentials, which includes the https URL using the token obtained
 from the GitHub application or secret attached to the repo CR.
 
+{{< hint info >}} For compatibility, the [Git
```

Comment:
i don't think we need multiple hints which is close to the other one and the info is a bit confusing for the end user... but no worries let's merge this and i'll rephrase that thereafter.

![image](https://github.com/openshift-pipelines/pipelines-as-code/assets/98980/f7c91684-485d-451f-8a99-74f9dc0d5e06)